src/com/company/problem_statement_3.java

```diff
@@ -0,0 +1,144 @@
+package com.company;
```

Comment:
Please read the problem statement again, it is incomplete. 
Searching, Sorting and Direction handling is not yet done in the code, please take care of that part as well.

Reply:
I have now handled the queries related to search, sort(orderby) and direction.